src/index.ts

```diff
@@ -15,14 +15,19 @@ export const main: HttpFunction = async (req, res) => {
         const client = ldap.createClient({ url: process.env.LDAP_URL as string })
         const password = process.env.LDAP_PASSWORD!.replaceAll("\n", "")
         await client.bind(process.env.LDAP_DN as string, password)
+        
         console.log("client binded")
 
-        const barcode = req.query["barcode"]
-        console.log("looking up barcode: ", barcode)
-        const results = await client.searchReturnAll("ou=people,dc=umd,dc=edu", {
-            filter: `(&(objectClass=*)(umLibraryBarcode=${barcode}))`,
-            scope: "sub",
-        })
+        const results = "barcode" in req.query ? 
```

Comment:
a) use an if statement instead of ternary, is pretty gross
b) a lot of code duplication - you only really need like a variable `search` which you set to `umLibraryBarcode=...` or `employeeNumber=...`